metagpt/actions/action_node.py

```diff
@@ -81,7 +81,7 @@ def __init__(
         instruction: str,
         example: Any,
         content: str = "",
-        children: dict[str, "ActionNode"] = None,
+        children: Optional[Dict[str, "ActionNode"]] = None,
```

Comment:
which tool do you used to find the increment content of the PR?

Reply:
I have been using `mypy` and a bit of `pyright`. I started the thing cause I faced a couple of exceptions when trying to do a simple experiment. So I started with the files that were close to the issues.